constant/constant.go

```diff
@@ -0,0 +1,18 @@
+package constant
+
+const (
```

Comment:
move  it to proper domain

Reply:
moved the constant to item and view package